src/components/Capture/capture.js

```diff
@@ -59,6 +59,12 @@ class Capture extends Component {
     actions.createCapture({method, capture, maxCaptures: this.maxAutomaticCaptures})
   }
 
+  createLivenessVideo(liveness, url) {
+    const payload = {liveness, url}
```

Comment:
what does `liveness` mean here?

Reply:
It's just a parameter I pass to the capture, so that the `Confirm` screen knows it has to render the video tag instead of the `CaptureViewer` component.